apps/system-e2e/src/tests/judicial-system/custody-prosecutor.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { BrowserContext, expect, test } from '@playwright/test'
+
+import { urls } from '../../support/urls'
+import { judicialSystemSession } from '../../support/session'
+
+test.use({ baseURL: urls.judicialSystemBaseUrl })
+
+test.describe('Custody Prosecutor', () => {
+  let context: BrowserContext
+
+  test.beforeAll(async ({ browser }) => {
+    context = await judicialSystemSession({
+      browser,
+    })
+  })
+
+  test.afterAll(async () => await context.close())
+
+  test('should submit a custody request to court', async () => {
+    const page = await context.newPage()
+
+    // Case list
+    page.goto('/krofur')
+    await page.getByRole('button', { name: 'Nýtt mál' }).click()
+    await page.getByRole('menuitem', { name: 'Gæsluvarðhald' }).click()
+    await expect(page).toHaveURL('/krafa/ny/gaesluvardhald')
+
+    // New custody request
+    await expect(
+      page.getByRole('heading', { name: 'Gæsluvarðhald' }),
+    ).toBeVisible()
+    await page.locator('input[name=policeCaseNumbers]').fill('007-2023-000001')
+    await page.getByRole('button', { name: 'Skrá númer' }).click()
+    await page.getByRole('checkbox').first().check()
+    await page.locator('input[name=accusedName]').fill('Jón Jónsson')
+    await page.locator('input[name=accusedAddress]').fill('Einhversstaðar 1')
+    await page.locator('#defendantGender').click()
+    await page.locator('#react-select-defendantGender-option-0').click()
+    await page
+      .locator('input[id=react-select-defenderName-input]')
+      .fill('Saul Goodman')
+    await page.locator('#react-select-defenderName-option-0').click()
+    await page
+      .locator('input[name=defenderEmail]')
+      .fill('jl-auto-defender@kolibri.is')
```

Comment:
Maybe use `@island.is` or `@devland.is` instead?

Reply:
We will actually get an email to this address, which contents we want to be able to verify from time to time. The email address "jl@kolibri.is" is white listed by SES on dev.